.github/workflows/ci.yaml

```diff
@@ -403,7 +403,7 @@ jobs:
 
       - name: Test
         run: |
-          ./project/scripts/sbt ";scala3-bootstrapped/compile ;scala3-bootstrapped/test;sjsSandbox/run;sjsSandbox/test;sjsJUnitTests/test;sjsCompilerTests/test ;sbt-test/scripted scala2-compat/* ;configureIDE ;stdlib-bootstrapped/test:run ;stdlib-bootstrapped-tasty-tests/test"
+          ./project/scripts/sbt ";dist/pack ;scala3-bootstrapped/compile ;scala3-bootstrapped/test;sjsSandbox/run;sjsSandbox/test;sjsJUnitTests/test;sjsCompilerTests/test ;sbt-test/scripted scala2-compat/* ;configureIDE ;stdlib-bootstrapped/test:run ;stdlib-bootstrapped-tasty-tests/test"
```

Comment:
is this just needed for the bootstrapCmdTests? If so, couldn't these tests run dist/pack themselves so we don't need to remember to do pack by hand before?

Reply:
I'm afraid that tests that need packing are coming from `scala3-bootstrapped/test`. I know they could and probably should be in a different configuration, but I didn't create them. Actually, I saw that these tests were silently failing for a long long time (e. g. https://github.com/lampepfl/dotty/runs/4470356079?check_suite_focus=true#step:9:764) and turned them in #14073 not knowing that they will do such turmoil in the whole CI (most of the checks are not run on a simple PR). Maybe we could move them to a separate isolated configuration for that, though @philwalk can bring more insight on that.

Here is a screenshot from old action where there are millions of stderrs printed out, yet the overall process was succeeding
![obraz](https://user-images.githubusercontent.com/32793002/146552747-bca09560-8cae-4d47-ba03-a5b8cf6e202e.png)

Reply:
In that case we should make sure dist is run automatically when we run `scala3-bootstrapped/test`, for example using `dependsOn(dist/pack)` a bit  like what we do for the coursier tests: https://github.com/lampepfl/dotty/blob/7a473349b3fb64e7c6c9dfcfb02758f4a2e3756c/project/Build.scala#L777